src/Engine/ProtoCore/DSASM/Executive.cs

```diff
@@ -3902,7 +3902,7 @@ private void GE_Handler(Instruction instruction)
                 {
                     double lhs = opdata2.IsDouble ? opdata2.DoubleValue : opdata2.IntegerValue;
                     double rhs = opdata1.IsDouble ? opdata1.DoubleValue : opdata1.IntegerValue;
-                    opdata2 = StackValue.BuildBoolean(MathUtils.IsGreaterThanOrEquals(lhs, rhs));
+                    opdata2 = StackValue.BuildBoolean(lhs >= rhs);
```

Comment:
I'm concerned that this loses the toleranced comparison.  If lhs was less than rhs but within the floating point tolerance, then this will change from true to false.

Reply:
Okay, just got to read the comments on the PR itself, and see why this is being done.